source/guides/stacks/mers.rst

```diff
@@ -417,4 +506,53 @@ using pre-trained models and sample video files.
        ./gst-video-analytics/samples/shell/console_measure_fps_cpu.sh $VIDEO_EXAMPLES_DIR/bolt-detection.mp4
 
 
+Adding AOM support
+******************
+
+The current version of |MERS| does not include the `Alliance for Open Media
+<https://aomedia.org/>`_ Video Codec (AOM). AOM can be built from source on an
+individual basis.
+
+To add AOM support to the |MERS| image:
+
+
+#. The following tools are needed to add AOM support to |MERS|: **docker, git,
+   patch**. On |CL| these can be  installed with the commands below. For other
+   operating systems, install the appropriate packages. 
+
+   .. code:: bash
+
+      sudo swupd bundle-add containers-basic dev-utils
+
+
+#. Clone the Intel Stack repository from GitHub:
+
+   .. code:: bash
+
+      git clone https://github.com/intel/stacks.git 
+
+#. Navigate to the directory for the |MERS| image.
+
+   .. code:: bash 
+
+      cd stacks/mers/clearlinux/
+
+#. Apply the patch to the :file:`Dockerfile`.
+
+   .. code:: bash
+
+      patch -p1 < aom-patches/stacks-mers-v2-include-aom.diff
+
+#. Use the :command:`docker build` command to build a local copy of the
+   MeRS container image tagged as *aom*.
+
+   .. code-block:: bash
+
+      docker build --no-cache -t sysstacks/mers-clearlinux:aom .
+
+Once the build has completed successfully, the local image can be used
+following the same steps in this tutorial by substituting the image name with
+*sysstacks/mers-clearlinux:aom*
+
+
 **OpenVINO is a trademark of Intel Corporation or its subsidiaries.**
```

Comment:
Update disclaimer to be "Intel, Xeon, OpenVINO, and the Intel logo are trademarks of Intel Corporation or its subsidiaries."